app/main.py

```diff
@@ -194,7 +190,8 @@ async def to_approve():
     return needs_approval
 
 
-@repeat_every(seconds=60 * 60)
+@app.on_event("startup")
+@repeat_every(seconds=60 * 4)
```

Comment:
I can see this is where you updated the timer to run every 4 minutes.

Reply:
lets remove the API call after this in a future PR.